src/components/VideoPlayerControls/VideoPlayerControlsPip.vue

```diff
@@ -35,12 +35,15 @@ export default {
       pip: (state) => state.pip,
       video: (state) => state.video,
     }),
+    ...mapState('Params', {
+      viewer: (state) => state.viewer
+    })
   },
   methods: {
     togglePip() {
       if (
         !this.pip &&
-        this.video.srcObject &&
+        (this.video.srcObject || this.viewer.drm) &&
```

Comment:
@gregdolby for tomorrow - when DRM is enabled, the user shouldn't be able to see the pip option (or toggle it on).  

I could be reading this wrong but looks like if drm is on and srcObject isn't null, we'll allow pip?